### DISCUSSION TITLE
Proof of Attendence

### DISCUSSION THREAD

**Discussion by bentotype** [community/community#130518] [GitHub Education]
### Hi everyone,

Hi Everyone,
I am having trouble uploading a document to show proof of my attendance at my University. My student ID doesn't have the date and the university hasn't sent me any letters with all of the requirements Github needs.

**Comment by davevad93** [comment]
Hi @bentotype ,

An alternative would be to upload a photo or screenshot of your most recent exam results with a date in `DD-MM-YYYY` or `YYYY-MM-DD` format, that worked for me.

**Comment by bentotype** [comment]
Hi @davevad93,
Is there another method because I can't use that method either. All of my exams in the last semester were online.

**Reply by davevad93** [reply]
Well, I guess if you have taken the exams online you have a record of your results somewhere, a screenshot of that would be enough. For instance I've uploaded a screenshot of my exam results which my institute publishes on their website.
If you can't do that either, the only workaround I can think of would be to upload a screenshot of a document like a letter or an email signed by the principal or a teacher from your school certifying that you are currently studying there.

**Comment by Anamicca23** [comment]
Hey @bentotype , welcome to GitHub Community!
To support your application, you can provide any of the following documents:
- A photo of your school ID showing your current enrollment date
- Your academic transcript
- A letter verifying your school affiliation or enrollment status
- Your class schedule
- screenshot of comments date in online class

If these options don't work out, you can always reach out to GitHub Education Support by submitting a ticket [here](https://support.github.com/contact/education).

Hope this information is helpful for you!

For more details, you can refer to the official documentation [here](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/apply-for-a-student-developer-pack).


plzz mark as " Answered" if it worked.

**Comment by bentotype** [comment]
Hello @Anamicca23,
I tried all of it including a letter from my school that proves attendance. It is also impossible to email support because they removed the option to ask about anything related to Github education.
